<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceEditor</class>
 <widget class="QWidget" name="ResourceEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="ui_splEditor">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="ui_editorLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QsciScintilla" name="qsciResourceEditor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="ui_widFind" native="true">
         <layout class="QHBoxLayout" name="ui_findLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="ui_leFindText">
            <property name="placeholderText">
             <string>Find text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ui_pbFind">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Find</string>
            </property>
            <property name="shortcut">
             <string>F3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ui_pbFindBackwards">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Find backwards</string>
            </property>
            <property name="shortcut">
             <string>Shift+F3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ui_leReplaceText">
            <property name="placeholderText">
             <string>Replace Text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ui_pbReplace">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Replace</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+=</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ui_widPlugins" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="ui_splPlugins">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/imgs/save.png</normaloff>:/imgs/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/imgs/find.png</normaloff>:/imgs/find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionPlugins">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/imgs/appicon.png</normaloff>:/imgs/appicon.png</iconset>
   </property>
   <property name="text">
    <string>Plugins</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>qsciResourceEditor</tabstop>
  <tabstop>ui_leFindText</tabstop>
  <tabstop>ui_leReplaceText</tabstop>
 </tabstops>
 <resources>
  <include location="bgcms.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>ResourceEditor</receiver>
   <slot>saveResource()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pbFind</sender>
   <signal>clicked()</signal>
   <receiver>ResourceEditor</receiver>
   <slot>find()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pbFindBackwards</sender>
   <signal>clicked()</signal>
   <receiver>ResourceEditor</receiver>
   <slot>findBackwards()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFind</sender>
   <signal>toggled(bool)</signal>
   <receiver>ui_widFind</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pbReplace</sender>
   <signal>clicked()</signal>
   <receiver>ResourceEditor</receiver>
   <slot>replace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPlugins</sender>
   <signal>toggled(bool)</signal>
   <receiver>ui_widPlugins</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveResource()</slot>
  <slot>toggleFind(bool)</slot>
  <slot>find()</slot>
  <slot>findBackwards()</slot>
  <slot>replace()</slot>
 </slots>
</ui>
